Dockerfile

```diff
@@ -1,20 +1,22 @@
-FROM ubuntu:xenial
-
-RUN apt-get -y update \
-    && apt-get -y install \
-    curl wget
+FROM alpine:3.6
 
+ENV ALPINE_GLIBC_VERSION 2.26-r0
 ENV HELM_VERSION 2.6.2
 
+RUN apk update && apk --no-cache add ca-certificates openssl curl wget git bash \
```

Comment:
We probably don't actually need wget and curl - using curl -o should suffice to replace wget.

Reply:
Replaced wget with curl calls